package.json

```
@@ -2,16 +2,17 @@
   "name": "pdf-to-markdown",
   "version": "0.1.1",
   "description": "A PDF to Markdown Converter",
-  "main": "main.js",
+  "main": "src/javascript/pdf2md.js",
```

Surely this should be...
```suggestion
  "main": "dist/pdf2md.js",
```